src/Neo/SmartContract/StorageKey.cs

```diff
@@ -255,30 +273,26 @@ public StorageKey()
         /// </summary>
         /// <param name="id">Contract Id</param>
         /// <param name="cache">The cached byte array. NOTE: It must be read-only and can be modified by the caller.</param>
-        private StorageKey(int id, byte[] cache)
+        private StorageKey(int id, ReadOnlySpan<byte> cache)
         {
-            _cache = cache;
+            // DO NOT CHANGE OR ELSE "Create" WILL HAVE PROBLEMS
+            _cache = cache.ToArray(); // Make a copy
             Id = id;
-            Key = cache.AsMemory(sizeof(int));
+            Key = _cache[sizeof(int)..]; // "Key" init makes a copy already.
```

Comment:
It's created by static method, who don't change anything after the creation, so I still seeing this copy a waste of time

Reply:
The constructor is private, and is constructed in static methods in this class, no security issue here, just an unnecessary copy, other can be discussed, this is unneeded